native/packages/react-native-bpk-component-image/src/BpkImage-test.common.js

```diff
@@ -0,0 +1,124 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import renderer from 'react-test-renderer';
+
+import BpkImage from './BpkImage';
+
+const commonTests = () => {
+  describe('BpkImage', () => {
+    it('should render correctly', () => {
+      const tree = renderer.create(<BpkImage />).toJSON();
```

Comment:
Could you add `alt` and `source` to this test too? We try to make sure all tests include the required props, as this matches our philosophy of not supporting/considering components with missing required props.